[2021-05-25 20:26:06,628] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T20:00:07.052122+00:00 [queued]>
[2021-05-25 20:26:07,021] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T20:00:07.052122+00:00 [queued]>
[2021-05-25 20:26:07,021] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 20:26:07,022] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-05-25 20:26:07,022] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 20:26:07,145] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Get_COVID19_Cases_For_Jordan_2020ToNow> on 2021-05-25T20:00:07.052122+00:00
[2021-05-25 20:26:07,183] {standard_task_runner.py:52} INFO - Started process 979 to run task
[2021-05-25 20:26:07,297] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'covid19_cases_Jordan', 'Get_COVID19_Cases_For_Jordan_2020ToNow', '2021-05-25T20:00:07.052122+00:00', '--job-id', '88', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/workflow.py', '--cfg-path', '/tmp/tmp4wv360qt', '--error-file', '/tmp/tmpt5e4srw_']
[2021-05-25 20:26:07,346] {standard_task_runner.py:77} INFO - Job 88: Subtask Get_COVID19_Cases_For_Jordan_2020ToNow
[2021-05-25 20:26:07,893] {logging_mixin.py:104} INFO - Running <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T20:00:07.052122+00:00 [running]> on host 4e0af5d83701
[2021-05-25 20:26:08,569] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=waed
AIRFLOW_CTX_DAG_ID=covid19_cases_Jordan
AIRFLOW_CTX_TASK_ID=Get_COVID19_Cases_For_Jordan_2020ToNow
AIRFLOW_CTX_EXECUTION_DATE=2021-05-25T20:00:07.052122+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-25T20:00:07.052122+00:00
[2021-05-25 20:52:40,393] {local_task_job.py:77} ERROR - Received SIGTERM. Terminating subprocesses
[2021-05-25 20:52:40,499] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 979
[2021-05-25 20:52:40,507] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
